### PR TITLE
feat: Add tablet rotate transformation

### PR DESCRIPTION
**Pull request by jp7677**
Same as the other tablet PRs, based on @Consolatis work here https://github.com/labwc/labwc/issues/1060#issuecomment-1786384495

### CODE REVIEW

File: include/config/rcxml.h [hunk 1]
```diff
@@ -83,6 +83,7 @@ struct rcxml {
 
 	/* graphics tablet */
 	struct tablet_config {
+		enum rotation rotate;
```
Comment (johanmalm) [inline include/config/rcxml.h:86]: Suggest `rotation` instead of `rotate`.... so `enum rotation rotation`

**Review by johanmalm — COMMENTED** [review]
All looks really tidy.
Just one minor suggestion.
I can't test it, but trust it works :smile:

File: include/config/rcxml.h (hunk 1 above)
Comment (jp7677) [reply include/config/rcxml.h]: Adjusted, thanks!

**Comment by jp7677** [comment]
> All looks really tidy. Just one minor suggestion. I can't test it, but trust it works 😄

Very cool!. And yes, the transformation works, using `rotate="180"` I can turn around my tablet and the cursor still follows my stylus moves ;)

**Comment by johanmalm** [comment]
With thanks.

**Pull request merged**